.github/workflows/release.yaml

```diff
@@ -36,10 +39,27 @@ jobs:
       - name: setup docker buildx
         run: docker buildx create --name conftestbuild --use
 
-      - name: push images
-        env:
-          VERSION: ${{ steps.get-version.outputs.VERSION }}
-        run: make push TAG=$VERSION
+      - name: Build and push Docker image
```

Comment:
Please add this to the PR workflow as well, with push set to false. That way we can know the container builds successfully before we try to tag a version of conftest.

Reply:
Hi @jalseth 
This is done now. Kindly review